src/main/java/com/faforever/client/mod/UploadUrlResponse.java

```diff
@@ -0,0 +1,8 @@
+package com.faforever.client.mod;
+
+import java.net.URI;
+import java.util.UUID;
+
+public record UploadUrlResponse(URI uploadUrl, UUID requestId) {
+
+} // todo: move to faf-java-commons
```

Comment:
Do we really want to move this? It will not be used by other java projects.

Reply:
I thought it will be a good idea, to do it some day. Because its the same code from `faf-java-api`. Literally copy/pasted it from there and added the `todo`

Reply:
Tested on `faforever.xyz` and the upload works.